Jamulus.pro

```diff
@@ -141,11 +143,22 @@ win32 {
     HEADERS += mac/activity.h
     OBJECTIVE_SOURCES += mac/activity.mm
     CONFIG += x86
-    QMAKE_TARGET_BUNDLE_PREFIX = net.sourceforge.llcon
+    QMAKE_TARGET_BUNDLE_PREFIX = io.jamulus
     QMAKE_APPLICATION_BUNDLE_NAME. = $$TARGET
 
+    #QMAKE_PROVISIONING_PROFILE = 
+    #QMAKE_DEVELOPMENT_TEAM = 
+
+    OSX_ENTITLEMENTS.files = Jamulus.entitlements
+    OSX_ENTITLEMENTS.path = Contents/Resources 
+    QMAKE_BUNDLE_DATA += OSX_ENTITLEMENTS
+    
     macx-xcode {
         QMAKE_INFO_PLIST = mac/Info-xcode.plist
+        XCODE_ENTITLEMENTS.name = CODE_SIGN_ENTITLEMENTS
+        XCODE_ENTITLEMENTS.value = Jamulus.entitlements
+        QMAKE_MAC_XCODE_SETTINGS += XCODE_ENTITLEMENTS
+        ICON = mac/mainicon.icns
```

Comment:
Is this a "bug fix" (Icon was not part of XCode produced app?)

Reply:
Xcode cannot be used to release the app at this time. One of the issues is the icon is not pulled in. One of the postings I read was to use this, but it has not changed anything. I will keep looking for the right change here.